includes/functions.php

```diff
@@ -1574,6 +1574,11 @@ function add_to_outbox( $data, $activity_type = 'Create', $user_id = 0, $content
 		return false;
 	}
 
+	// If the user is disabled, use the blog user.
+	if ( is_user_disabled( $user_id ) && ! is_user_disabled( Actors::BLOG_USER_ID ) ) {
+		$user_id = Actors::BLOG_USER_ID;
+	}
```

Comment:
we also need the check the case `if ( is_user_disabled( $user_id ) && is_user_disabled( Actors::BLOG_USER_ID ) )` to bail out then!?!